package.json

```diff
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "mkdirp": "0.3.5",
-    "nodeunit": "0.8.0",
+    "nodeunit": "0.8.2",
```

Comment:
upgraded nodeunit because nodeunit 0.8.0 was running into a common issue that is fixed in this version